manifests/release/kubevirt.yaml.in

```diff
@@ -1,4 +1,84 @@
 # RBAC
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kubevirt.io:admin
+  labels:
+    kubevirt.io: ""
+rules:
+  - apiGroups:
+      - subresources.kubevirt.io
+    resources:
+      - virtualmachines/console
+      - virtualmachines/vnc
+    verbs:
+      - get
+  - apiGroups:
```

Comment:
The admin role will also need to be able to view ConfigMaps in the `kube-system` namespace.

https://github.com/kubevirt/kubevirt/pull/804 adds a controller that uses a ListerWatcher.

Reply:
good point

This is a little tricky though because of how roles bind to users.  Someone might want to bind the admin role to user (myuser) in a single namespace (mynamespace). In this instance myuser wouldn't have access to the configmap in kube-system.

I'll have to add a 4th role to give access to the configmap.  I'll call it kubevirt.io:runtime-config, or something like that.